docs/development/releases/README.md

```diff
@@ -0,0 +1,44 @@
+# Releases
+
+This page summarizes all relevant information about releases.
+
+## Distribution
+
+OpenProject is distributed by [various means](../../installation-and-operations/installation/). Manual installation based on the code in GitHub is possible but not supported.
+
+## Release notes
+
+You can find the release notes for major stable releases [here](../../release-notes/).
+
+## Versioning and tags
+
+### General considerations
+
+* See the [Roadmap](https://community.openproject.com/projects/openproject/roadmap) for the overview of the current stable release version and upcoming stable releases
+* Releases are defined by a version (number) which exists as tags on predefined development trees (see the branches on [github](https://github.com/opf/openproject/releases)).
+
+### Semantic versioning
```

Comment:
This could incorporate (or link to) the branching model. I think these two things belong together. WDYT? https://www.openproject.org/docs/development/git-workflow/#branching-model

Reply:
You were right, the overlap with the branching model description was large. I therefore moved that part over and then rewrote the release documentation even more. 

With what it is now, it is less a documentation for developers and more one for administrators. I think it should be moved but I can't pinpoint a better place. Do you have an idea @oliverguenther ?

Reply:
The same is true for data flow and the other recent additions. It's a technical documentation, but not necessary directed towards developers.